### PR TITLE
Fixed: Improve help text for download client Category

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
title
- don't mention optional for download client categories (see also enabling show unknown as default)
- add forced warning for qbit
#### Todos
- Tests
- [x] Wiki Updates

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public DelugeSettings()
         [FieldDefinition(4, Label = "Password", Type = FieldType.Password, Privacy = PrivacyLevel.Password)]
         public string Password { get; set; }
 
-        [FieldDefinition(5, Label = "Category", Type = FieldType.Textbox, HelpText = "Adding a category specific to Sonarr avoids conflicts with unrelated downloads, but it's optional")]
+        [FieldDefinition(5, Label = "Category", Type = FieldType.Textbox, HelpText = "Adding a category specific to Sonarr avoids conflicts with unrelated downloads")]
```
Comment (markus101) [inline src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs:47]: Why though? It's still optional.

Comment (bakerboy448) [reply src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs:47]: keeping it in means users then are confused why they have random downloads showing up given showunknown is enabled which solved the opposite problem of downloads 'disappearing' 

I guess `avoids conflicts` is not clear enough?

I think regardless it does need some sort of reword

Comment (markus101) [reply src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs:47]: Maybe: `Adding a category specific to Sonarr avoids conflicts with non-TV Series downloads. Using a category is optional, but strongly recommended.` still optional, but now recommended.

Comment (bakerboy448) [reply src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs]: pushed as `Adding a category specific to Sonarr avoids conflicts with unrelated non-Sonarr downloads. Using a category is optional, but strongly recommended.`

**Pull request merged**